app/models/episode_story_handler.rb

```diff
@@ -33,13 +33,21 @@ def update_from_story!(story)
 
   def update_from_story(story)
     self.story = story
-    episode.prx_uri = story.links['self'].href
+    episode.prx_uri = clean_prx_uri(story.links['self'].href)
 
     update_attributes
     update_audio
     update_image
   end
 
+  def clean_prx_uri(url)
+    result = url
+    if result && !result.match(/authorization/)
+      result = result.gsub('/', '/authorization/')
```

Comment:
this line replaces every `/` in the `url` with `/authorization/` - that isn't what I would expect a method called `clean_prx_uri` to do. Likewise, the name of this PR is "not with authorization in it" and this method is *adding* the `authorization`.

So I'm confused.

Reply:
me too, not sure how my test passed locally. updating.

Reply:
looks like `guard` was misbehaving - ok, fixing.